### ISSUE TITLE
Create the UI for the address book

### ISSUE THREAD

**Issue by cBridges851** [cBridges851/tkinter-address-book#1]
A UI needs to be created in Tkinter. There must be boxes for the user's forename, surname, first line of address, second line of address, town/city, county, country, postcode, phone number and email address (the fields that are in the database). This would be part of a form where they can make inputs.

There would also need to be a page where the user is able to search for a contact. This would contain an entry box for the value they wish to find, and a drop box for the information they are providing.

There needs to be a landing page so the user can select which part of the program they wish to access.

To summarise:
- Landing page (root)
- Input a contact
- Find a contact
